build.gradle

```diff
@@ -0,0 +1,47 @@
+
+plugins {
+    id 'java-gradle-plugin'
+    id 'com.gradle.plugin-publish' version '0.10.1' 
+    id 'maven-publish'
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile(group: 'org.cyclonedx', name: 'cyclonedx-core-java', version: '2.1.1') {
+        // gradle-api already includes an slf4j binding
+        exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j-impl'
+    }
+    compile 'commons-codec:commons-codec:1.13'
+    compile 'commons-io:commons-io:2.6'
+    compile 'org.apache.maven:maven-core:3.5.0'
+}
+
+group = 'com.cyclonedx'
+version = '1.1.2-SNAPSHOT'
```

Comment:
VERY IMPORTANT  This 'version' field needs to be manually maintained in sync with pom.xml

Reply:
I'll likely add this to the pom so that its automatic